docs/releases/1.11.txt

```diff
@@ -179,6 +179,8 @@ Management Commands
 * The new :option:`loaddata --exclude` option allows excluding models and apps
   while loading data from fixtures.
 
+* The :djadmin:`runserver` Web server supports HTTP 1.1.
```

Comment:
Speaking of that, the selenium tests aren't working on Chrome for me after this change.
```
selenium.common.exceptions.WebDriverException: Message: session not created exception
from unknown error: failed to close UI debuggers
```